shared_fields_common.view.lkml

```diff
@@ -366,6 +371,83 @@ view: shared_fields_common {
     group_label: "Marketing"
   }
 
+  dimension: marketing_date {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',1) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_ministry {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',2) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_team {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',3) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_campaign_id {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',4) ;;
+    group_label: "Marketing"
+    drill_fields: [marketing_drills*]
+
+  }
+
+  dimension: marketing_sequence {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',5) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_cta {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',6) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_platform {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',7) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_sender_placement {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',8) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_lang {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',9) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_audience {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',10) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_sub_audience {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',11) ;;
+    group_label: "Marketing"
+  }
+
+  dimension: marketing_ad_type {
+    type: string
+    sql: SPLIT_PART(${TABLE}.mkt_campaign,'_',12) ;;
+    group_label: "Marketing"
+  }
+
```

Comment:
I wasn't sure whether or not the tables referenced by the other view files that import `shared_fields` had these `${TABLE}.mkt_*` columns that you've built these new dimensions on (in L374-L446 above).

Those other views/tables are:
- clicks.view.lkml (derived.clicks)
- searches.view.lkml (derived.searches)

I checked those tables, and it appears that all the tables referenced _do_ have those columns; so there should be no need to extend those dimensions with a `hidden: yes` to avoid a column referencing error.

Reply:
Not a problem. Thanks for checking.